package.json

```
@@ -0,0 +1,67 @@
+{
+  "name": "stylelint-create",
+  "version": "0.0.1",
+  "description": "Initialization wizard for Stylelint, a linter that helps you avoid errors and enforce conventions in your styles.",
+  "homepage": "https://github.com/stylelint/stylelint-create#readme",
+  "bugs": {
+    "url": "https://github.com/stylelint/stylelint-create/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/stylelint/stylelint-create.git"
+  },
+  "license": "MIT",
+  "author": "stylelint",
+  "main": "create-stylelint.mjs",
```

[trivial] Perhaps `main` is unnecessary because this package is pure ESM.
```suggestion
  "main": "create-stylelint.mjs",
```

---

Removing this causes triggers the [`require-main`](https://npmpackagejsonlint.org/docs/rules/required-node/require-main) rule in `npm-package-json-lint`. Should I keep the `main` entry, or disable the rule?

---

OK. Let's keep it as-is.

---

If need be we can ignore this rule for this package by adding this to the `npmpackagejsonlint` config object:

```
"overrides": [
    {
      "rules": {
        "require-main": "off"
      }
    }
  ]
```